app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/WizNavStartActivity.java

```diff
@@ -216,19 +216,19 @@ private void displayErrorDialog(Throwable throwable) {
      * Resets the LRZ ID (TUM ID) to null.
      */
     private void resetLrzId() {
-        Utils.setSetting(this, Const.LRZ_ID, null);
+        Utils.setSetting(this, Const.LRZ_ID, "");
     }
 
     /**
      * Resets the TUMonline access token to null.
```

Comment:
The documentation here and above is now incorrect.
I'd just remove it for those appropriately named one line functions...